BostonData/Intents/__init__.py

```diff
@@ -0,0 +1 @@
+__all__ = ["GetAddressIntent", "SetAddressIntent", "TrashDayIntent", "WorkZonesOnAnyStreetIntent", "WorkZonesOnMyStreetIntent"]
```

Comment:
It doesn't look like these are being used at all

Reply:
That was left over from me messing with sarat's pull. I'll fix this up and continue testing it.